Dockerfile.run

```diff
@@ -1,5 +1,12 @@
 FROM busybox:ubuntu-14.04
 
+# Avoid running as root within the container
+RUN groupadd -r user && useradd -r -g user user
```

Comment:
Unfortunately, it seems that `groupadd` is not available under busybox, and so this specification is pointless.

![image](https://cloud.githubusercontent.com/assets/4713937/26537680/50849914-43f6-11e7-8780-94016e39ffc0.png)